File: src/main/java/exam/paperContext/userInterface/query/BlankQuizResponse.java
package exam.paperContext.userInterface.query;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Setter
@Getter
public class BlankQuizResponse {
    private String id;
    private String number;
    private int score;
    private String content;
    private String referenceAnswer;
    private LocalDateTime createTime;
    private LocalDateTime lastUpdateTime;
    private LocalDateTime removeTime;
}

File: src/main/java/exam/blankquizcontext/userinterface/presentation/BlankQuizResponse.kt
package exam.blankquizcontext.userinterface.presentation

import java.time.LocalDateTime

data class BlankQuizResponse(var id: String?,
                             var number: String?,
                             var content: String?,
                             var referenceAnswer: String?,
                             var removed: Boolean,
                             var createTime: LocalDateTime?,
                             var lastUpdateTime: LocalDateTime?,
                             var removeTime: LocalDateTime?) {
    constructor() : this(null, null, null, null,
            false, null, null, null)
}
